base/compiler/ssair/inlining.jl

```diff
@@ -306,6 +306,19 @@ function finish_cfg_inline!(state::CFGInliningState)
     end
 end
 
+@noinline function method_name(m)
+    m = m isa LineInfoNode ? m.method : m
+    m = m isa MethodInstance ? m.def : m
+    m = m isa Method ? m.name : m
+    m = m isa Symbol ? m : Symbol("")
+    return m
+end
```

Comment:
At the moment, I don't recall why I have both this and the Base.StackTraces version.